R/batches.R

```diff
@@ -1,22 +1,19 @@
-addBatch <- function (ds, ..., savepoint=TRUE, autorollback=savepoint, strict=TRUE, body=list(...)) {
+addBatch <- function (ds, ..., savepoint, autorollback, strict=TRUE, body=list(...)) {
```

Comment:
This function is not exported, so remove the arguments here, and handle the removal/deprecation warning in the calling function(s).